corehq/apps/export/templates/export/commcare_analytics.html

```diff
@@ -0,0 +1,13 @@
+{%  extends 'hqwebapp/base_section.html' %}
+{% load crispy_forms_tags %}
+{% load i18n %}
+{% load compress %}
+{% load hq_shared_tags %}
+
+{% block page_content %}
+  {% initial_page_data 'analytics_redirect_url' analytics_redirect_url %}
```

Comment:
Don't need this since you're not accessing this value via javascript.

Reply:
Right! Forgot to remove that when I deleted the js file